README.md

```diff
@@ -593,6 +593,20 @@ This is the `forge:` key in r10k, it accepts a hash that contains settings for d
     }
 ```
 
+##### `deploy_settings`
+This is the `deploy:` key in r10k, it accepts a hash that contains setting that control how r10k code deployments behave.
```

Comment:
Provide a link to https://docs.puppet.com/pe/latest/r10k_custom.html#deploy here.